recipe/meta.yaml

```diff
@@ -56,15 +56,15 @@ requirements:
    - python-lsp-server >=1.7.1,<1.8.0
    - pyxdg >=0.26  # [linux]
    - pyzmq >=22.1.0
-   - qdarkstyle >=3.0.2,<3.1.0
+   - qdarkstyle >=3.0.2,<3.2.0
    - qstylizer >=0.2.2
    - qtawesome >=1.2.1
-   - qtconsole >=5.4.0,<5.5.0
+   - qtconsole >=5.4.2,<5.5.0
    - qtpy >=2.1.0
    - rtree >=0.9.7
    - setuptools >=49.6.0
    - sphinx >=0.6.6
-   - spyder-kernels >=2.4.2,<2.5.0
+   - spyder-kernels >=2.4.3,<2.5.0
    - textdistance >=4.2.0
    - three-merge >=0.1.1
    - watchdog >=0.10.3
```

Comment:
```
    # This is here to workaround an inconsistency with flake8
    # See: https://github.com/conda-forge/flake8-feedstock/issues/52
   - importlib-metadata <4.3 # [py<38]
```
Can be removed as we do not build for py37

Reply:
removed